annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -409,7 +409,7 @@ private String resolveHgvspShort() {
                     }
                 }
             }
-            else if (canonicalTranscript.getHgvsc() != null) {
+            else if (canonicalTranscript.getHgvsc() != null && !canonicalTranscript.getConsequenceTerms().get(0).equals("inframe_insertion")) {
```

Comment:
What about inframe deletions or missense? Maybe change this logic to look for the cases that are being used inside. Add those here instead so that it only goes in that block for splice site stuff.